koku/api/provider/models.py

```diff
@@ -181,7 +181,7 @@ def save(self, *args, **kwargs):
 
             LOG.info(f"Starting data ingest task for Provider {self.uuid}")
             # Start check_report_updates task after Provider has been committed.
-            transaction.on_commit(lambda: check_report_updates.delay(provider_uuid=self.uuid))
+            check_report_updates.s(provider_uuid=self.uuid, queue_name=PRIORITY_QUEUE).set(queue=PRIORITY_QUEUE)
```

Comment:
We still need this transaction, else this won't work correctly.
```suggestion
            transaction.on_commit(lambda: check_report_updates.s(provider_uuid=self.uuid, queue_name="priority").set(queue="priority")
```

I think the circular import is due do us importing this queue name.